qcfractal/tests/test_collections.py

```diff
@@ -1020,6 +1021,31 @@ def test_view_download_mock(gradient_dataset_fixture, tmp_path_factory):
                 ds.download(verify=True)
 
 
+def test_gradient_dataset_plaintextview_write(gradient_dataset_fixture, tmpdir):
+    _, ds = gradient_dataset_fixture
+    ptl.collections.PlainTextView(tmpdir / "test.tar.gz").write(ds)
```

Comment:
Should we implement this similar to `ds.to_file("test.tar.gz", dtype="plaintext")`?